onadata/apps/api/management/commands/delete_users.py

```diff
@@ -0,0 +1,75 @@
+from django.contrib.auth.models import User
+from onadata.apps.logger.models import XForm
+from onadata.apps.logger.models import Project
+from onadata.apps.logger.models import Instance
+from django.db.utils import IntegrityError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = 'Delete users'
+
+    def add_arguments(self, parser):
+        parser.add_argument('user', nargs='*')
+
+    def handle(self, *args, **kwargs):
+        users = kwargs['user']
+        if users:
+            usernames = []
+            for user in users:
+                user_details = user.split(':')
+                import ipdb; ipdb.set_trace()
+                username = user_details[0]
+                email = user_details[1]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'email' is assigned to but never used](https://app.codacy.com/app/Ona/onadata/pullRequest?prid=4523955)